src/local/butler/scripts/migration/monorail_to_issuetracker_migrator.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Migrate Issue Ids from Monorail to Issue Tracker."""
+
+import csv
+
+from google.cloud import ndb
+
+from clusterfuzz._internal.datastore import data_types
+
+
+def execute(args):
+  """Query Testcases of a project, and update the bug_information
+  and/or group_bug_information fields to reflect the Issue Tracker issue
+  id rather than the Monorail issue id."""
+
+  issue_id_dict = get_monorail_issuetracker_issue_id_dictionary(args.file_loc)
+
+  testcases = []
+
+  for testcase in data_types.Testcase.query(
+      # only target testcases in single project
+      data_types.Testcase.project_name == args.project_name,):
+    if testcase.bug_information and issue_id_dict.get(testcase.bug_information):
+      testcase.bug_information = ndb.StringProperty(
```

Comment:
I don't think we need ndb.StringProperty() wrapping this? I'm also not aware if this actually works either.